src/app/src/main/resources/application.properties

```diff
@@ -14,4 +14,5 @@ server.compression.mime-types=text/html,text/xml,text/plain,text/css,text/javasc
 # Compress the response only if the response size is at least 1KB
 server.compression.min-response-size=1024
 server.servlet.session.timeout=120m
-logging.config=classpath:logback-spring.xml
\ No newline at end of file
+logging.config=classpath:logback-spring.xml
+spring.messages.encoding=UTF-8
```

Comment:
This is an opportunistic fix, by default *.property files are not UTF-8 encoded, adding this line ensures internationalisation is encoded in UTF-8.